golang/src/envx_counters/counters.go

```diff
@@ -79,48 +80,52 @@ func HitDelta(name string, delta int64) {
 	if gDisabled {
 		return
 	}
+
+	gLock.RLock()
+	if v := gStorage[name]; v != nil {
+		gLock.RUnlock()
+		atomic.AddInt64(v, delta)
+		return
+	}
+	gLock.RUnlock()
+	i := delta
 	gLock.Lock()
-	v := gStorage[name]
-	gStorage[name] = v + delta
+	gStorage[name] = &i
 	gLock.Unlock()
 }
 
 // Increment counter value with 1.
 func Hitf(format string, args ...interface{}) {
-	HitDeltaf(format, 1, args...)
+	HitDelta(fmt.Sprintf(format, args...), 1)
```

Comment:
Here and below in HitDeltaf you removed ``if gDisabled`` clause and now fmt.Sprintf() will be performed disregard of gDisabled value.

Reply:
Fair point, you are right.